src/Schema/Events/Search.ts

```diff
@@ -88,6 +88,86 @@ export interface SearchedWithNoResults {
   query: string
 }
 
+/**
+ * A user selects search pill
+ *
+ * This schema describes events sent to Segment from [[selectedPillFromSearch]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "selectedPillFromSearch",
+ *    context_module: "header",
+ *    context_owner_type: "home",
+ *    destination_owner_type: "search",
+ *    query: "andhuwe"
+ *    pill: "artwork"
+ *  }
+ * ```
+ */
+export interface SelectedPillFromSearch {
```

Comment:
**question**: @abhitip should we use the `tappedNavigationTab` event when clicking on pills, as it happens in [Eigen](https://github.com/artsy/eigen/blob/192d1aa85d7435406f5bb51213a8fe28fa3382ab/src/app/Scenes/Search/Search2.tsx#L206-L213)?

Reply:
What is `tappedNavigationTab`?

Reply:
Is it similar to - what pill was active when a user taps a result ?

Reply:
Oh, interesting, I guess we use `subject` to capture that displayed text of the navigation tab ([source](https://github.com/artsy/cohesion/blob/1dcd0355091b3af34d478b26c1a1c1e5bed089ae/src/Schema/Events/Tap.ts#L559-L567)). Used here [in Force too](https://github.com/artsy/force/blob/main/src/Apps/Search/Components/NavigationTabs.tsx#L78-L86).

So yeah, maybe we can reuse that event and drop the idea of typing the allowed "pills" which are more susceptible to change if it's the display name we pass through.

Reply:
Oh yes let's use the same event if possible! That naming is a little confusing with the "navigation" but it's okay